### PR TITLE
Update metrics.md

### PR DESCRIPTION
**Pull request by JamesNK**
cc @noahfalk

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/log-mon/metrics/metrics.md](https://github.com/dotnet/AspNetCore.Docs/blob/be62de62bc0cee3c09fe2570b96e088e760616a4/aspnetcore/log-mon/metrics/metrics.md) | [ASP.NET Core metrics](https://review.learn.microsoft.com/en-us/aspnet/core/log-mon/metrics/metrics?branch=pr-en-us-30091) |

### CODE REVIEW

**Review by noahfalk — APPROVED** [review]
👍

File: aspnetcore/log-mon/metrics/metrics.md [hunk 1]
```diff
@@ -88,15 +87,7 @@ Press p to pause, r to resume, q to quit.
...
 
-Run `WebMetric>dotnet-counters list` to show all available metrics.
-
-The following command shows the `Microsoft-AspNetCore-Server-Kestrel` meter with the `requests-per-second` and `total-requests` counters.
-
-```dotnetcli
-dotnet-counters monitor -n WebMetric --counters Microsoft.AspNetCore.Hosting[requests-per-second,total-requests]
-```
-
-For more information, see [dotnet-counters](/dotnet/core/diagnostics/dotnet-counters). To learn more about metrics in .NET, see [built-in metrics](/dotnet/core/diagnostics/available-counters).
```
Comment (JamesNK) [inline aspnetcore/log-mon/metrics/metrics.md:99]: I remove the link to the available counters page as it just shows event counters for now.

File: aspnetcore/log-mon/metrics/metrics.md [hunk 2]
```diff
@@ -88,15 +87,7 @@ Press p to pause, r to resume, q to quit.
         host=localhost,method=GET,port=5045,protocol=HTTP/1.1,ro           0                                            12
 ```
 
-Run `WebMetric>dotnet-counters list` to show all available metrics.
```
Comment (JamesNK) [inline aspnetcore/log-mon/metrics/metrics.md:91]: `dotnet-counters list` isn't very useful right now. Removed.

File: aspnetcore/log-mon/metrics/metrics.md [hunk 3]
```diff
@@ -88,15 +87,7 @@ Press p to pause, r to resume, q to quit.
...
-
-The following command shows the `Microsoft-AspNetCore-Server-Kestrel` meter with the `requests-per-second` and `total-requests` counters.
-
-```dotnetcli
-dotnet-counters monitor -n WebMetric --counters Microsoft.AspNetCore.Hosting[requests-per-second,total-requests]
-```
```
Comment (JamesNK) [inline aspnetcore/log-mon/metrics/metrics.md:97]: This kind of detail can be left out. People who want to know can look at the dotnet-counters detail page.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**